test/models/transforms/test_input.py

```diff
@@ -591,27 +597,25 @@ def test_chained_input_transform(self):
         tf = ChainedInputTransform(stz=tf1, pert=tf2)
         self.assertTrue(tf.is_one_to_many)
 
-    def test_round_transform(self):
-        for dtype in (torch.float, torch.double):
-            # basic init
-            int_idcs = [0, 4]
-            categorical_feats = {2: 2, 5: 3}
-            # test deprecation warning
-            with warnings.catch_warnings(record=True) as ws, settings.debug(True):
-                Round(indices=int_idcs)
-                self.assertTrue(
-                    any(issubclass(w.category, DeprecationWarning) for w in ws)
-                )
-            round_tf = Round(
-                integer_indices=int_idcs, categorical_features=categorical_feats
-            )
-            self.assertEqual(round_tf.integer_indices.tolist(), int_idcs)
-            self.assertEqual(round_tf.categorical_features, categorical_feats)
-            self.assertTrue(round_tf.training)
-            self.assertFalse(round_tf.approximate)
-            self.assertEqual(round_tf.tau, 1e-3)
-            self.assertTrue(round_tf.equals(Round(**round_tf.get_init_args())))
+    def test_round_transform_init(self):
```

Comment:
While we're at it ...
```suggestion
    def test_round_transform_init(self) -> None:
```